### ISSUE TITLE
English Language Descriptions of the Athena Algorithms 

### ISSUE THREAD

**Issue by HowardLee6** [AthenaModel/athena#3]
Description: In an effort to provide increased understanding of the computational underpinnings of the Athena Simulation, a series of explanatory ‘English language descriptions/translations of the Athena algorithms’ would be helpful.  Each of the major internal modules would benefit from a clarifying commentary in plain English, perhaps coupled with an outline of the social science theory that supports the module.  

The explanatory topics of interest are aligned below with the modeling areas as found in the Athena User's Guide:
- Physical Modeling Area
   - Security
   - Volatility
   - Proximity
- Politics Modeling Area
   - Actors
   - Strategy
   - Relationships
   - Support
   - Influence
   - Control
- Social Modeling Area
   - Belief Systems and Affinities
   - Mood
   - Attitudes: URAM (Unified Regional Attitude Model) and DAM (Driver Assessment Model)
   - Consumption of Goods
   - Unemployment effects
- Economics Modeling Area
   - GDP
   - Unemployment Rate
- Infrastructure Modeling Area
   - Goods Plants
- Information Modeling Area
   - Messaging

**Comment by bjkahovec** [comment]
I plan to write a plain English description of the Security and Volatility models as a proof-of-principle. If found to be useful, I will write natural language descriptions for each of the models listed above.
